src/Animated.test.js

```diff
@@ -0,0 +1,102 @@
+import { getNumberOfInitializedNodes_forTestsOnly as numberOfNodes } from './core/AnimatedNode';
+import Animated, { Easing } from './Animated';
+import AnimatedAlways from './core/AnimatedAlways';
+
+jest.mock('./ReanimatedEventEmitter');
+jest.mock('./ReanimatedModule');
+
+const { Value, timing, spring, decay } = Animated;
+expect.extend({
+  toAttachNodesProperly(animation) {
+    const transX = new Value(0);
+
+    const v = new AnimatedAlways(transX);
+    const initial = numberOfNodes();
+    v.__addChild(v);
+    const before = numberOfNodes();
+    const anim = animation.node(transX, animation.config);
+    anim.start();
+    const during = numberOfNodes();
+    anim.stop();
+    const after = numberOfNodes();
+    v.__removeChild(v);
+    const final = numberOfNodes();
+
+    const pass =
+      initial === final &&
+      after === before &&
+      during > after &&
+      initial === 0 &&
+      before === 2;
+    if (pass) {
+      return {
+        message: () =>
+          `expected ${
+            animation.name
+          }'s nodes to be attached and detached correctly`,
```

Comment:
Do we need "message" if `"pass: true"`? Is it being shown at all? I though the message is only needed when test fail so that we can tell what was wrong